integration/benchmarks/rax/run_benchmark.py

```
@@ -0,0 +1,740 @@
+                        info_dict[k] = info_idx[i+1]
+                elif isinstance(info_idx, dict):
+                    info_dict = {k.decode() if isinstance(k, bytes) else str(k): v for k, v in info_idx.items()}
+                else:
+                    raise ValueError(f"Unexpected FT.INFO output type: {type(info_idx)}")
+
+                raw_docs = info_dict.get("num_docs") or info_dict.get(b"num_docs", 0)
+                indexed_docs = int(raw_docs)
+                if indexed_docs >= num_docs:
+                    settled = True
+                    break
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)